### PR TITLE
Fixed sidebar scaling issues

### PR DESCRIPTION
**Pull request by morganbroome**
Changed height calculation to properly scale the sidebar length with window size

### CODE REVIEW

File: client/src/components/sidebar.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ import '../styles/styles.css';
 export default function SidebarComp() {
     return (
         <Sidebar className="sidebar">
-            <img style={{ "width": 100 + '%', "height": 25 + "%" }} src="emory.png"></img>
+            <img style={{ "width": 250 + 'px', "height": 230 + "px" }} src="emory.png"></img>
```
[the remark this answers is not in the document]
Comment (morganbroome) [reply client/src/components/sidebar.js:18]: I promise it looks better this way

**Review by thespych — APPROVED** [review]
looks good! smart fix

**Review by edong36 — APPROVED** [review]

File: client/src/styles/styles.css [hunk 2]
```diff
@@ -10,7 +10,7 @@
 
 .menu {
   width: 100%;
-  min-height: 75%;
+  min-height: calc(100% - 230px);
```
Comment (NamkhangNLe) [inline client/src/styles/styles.css:13]: Super smart actually.

**Review by MichaelEHammond — APPROVED** [review]
Looks great. Good catch with the min-height.

**Review by dsoman24 — APPROVED** [review]
LGTM! Lets get this fix in.

**Pull request merged**